packages/calling-component-bindings/src/handlers/createCommonHandlers.ts

```diff
@@ -262,6 +270,36 @@ export const createDefaultCommonCallingHandlers = memoizeOne(
     const onToggleScreenShare = async (): Promise<void> =>
       call?.isScreenSharingOn ? await onStopScreenShare() : await onStartScreenShare();
 
+    /* @conditional-compile-remove(raise-hands) */
+    const onRaiseHand = async (): Promise<void> => await call?.feature(Features.RaiseHand)?.raiseHand();
```

Comment:
Some indent issues here, would be nice to run rush lint: fix to fix all of these

Reply:
NVM, looks like the sentence wrap problem in my local editor